Car.cpp

```diff
@@ -0,0 +1,41 @@
+#include "Car.hpp"
+#include"Exceptions.hpp"
+#include <iostream>
+
+Car::Car()		  { std::cout << __FUNCTION__ << std::endl;
+                            velocity=0; }
+Car::~Car()		  { std::cout << __FUNCTION__ << std::endl; }
+
+void Car::turnLeft()      { std::cout << __FUNCTION__ << std::endl; }
+void Car::turnRight()     { std::cout << __FUNCTION__ << std::endl; }
+void Car::brake()
+{
+                            std::cout << __FUNCTION__ << std::endl;
```

Comment:
ale duże wcięcie. Jak już jest w kolejnej linijce to formatujcie ze standardowym 4 spacjowym wcięciem. Poniżej podobnie.